src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
```

Comment:
we can add a constructor to our class, so we won't need to set them
our object will be valid after creation